packages/venia-concept/src/components/Navigation/__tests__/categoryTree.spec.js

```diff
@@ -153,7 +153,7 @@ test('renders with product data', async () => {
         </MockedProvider>
     );
 
-    await wait();
+    await wait(500);
```

Comment:
added this to prevent flakiness, the test needed a little more time to resolve before the assertion. reran the tests in ci 3 times and did not see it come up again, but we may want to keep an eye on this one to see if it still causes issues. Saw the flakiness both locally as well as up in ci.

Reply:
I don't think this is the right solution. If `await wait()` isn't working, we need to find something else.

What we're trying to do here is wait for all promises to resolve, which may be several ticks. I noticed that other tests sometimes tell Jest to run timers and such, so that may be the key. We should find the minimal instruction and use it consistently.

Reply:
@zetlen here is the quick fix I put up for this, we can add to the issue more details if you have ideas for a better solution, but I wanted to unblock folks from its flakiness for now.

Reply:
There's a fix for this that seems to work well in #738. @sharkySharks if you can review that PR and merge it, this change might be unnecessary!